test/common/http/codec_client_test.cc

```diff
@@ -158,6 +158,7 @@ TEST_F(CodecClientTest, IdleTimerWithNoActiveRequests) {
   // Close the client and validate idleTimer is reset
   EXPECT_EQ(client_->numActiveRequests(), 0);
   client_->close();
+  // TODO: Use default connection mock handlers for raising events.
```

Comment:
Per the style guide, please put your github user name in these TODOs: `//TODO(ramaraochavali): ...`